src/components/Movies.js

```diff
@@ -0,0 +1,59 @@
+import "../App.css";
+import React from "react";
+
+
+class Movies extends React.Component {
+  constructor(){
+    super()
+
+    this.state ={
+      films:[],
+      currentFilms: null,
+    };
+  }
+
+  fetchFilms = () =>{
+    fetch("https://ghibliapi.herokuapp.com/films")
+    .then((res) => res.json())
+    .then((data)=> {
+      this.setState({
+        films: data,
+      });
+    }) ;
+  };
+  
+  componentDidMount = () =>{
+    this.fetchFilms();
+  }
+
+
+  handleDropDown = (event) =>{
+    let currentMovieObj = this.state.films.find((film) =>{
+      return film.title === event.target.value
+    })
+  }
```

Comment:
```suggestion
  handleDropDown = (event) =>{
    let currentMovieObj = this.state.films.find((film) =>{
      return film.title === event.target.value
    })
    this.setState({
      currentFilm: currentMovieObj,
    });
  }
```
your handleDropDown function was off to a great start! But you never ended up using your `currentMovieObj` variable. You needed to set it in state using `this.setState` so that you'd have access to it throughout the whole component and be able to render it to the screen using JSX in the render() method below.